### PR TITLE
Kay

### PR DESCRIPTION
**Pull request by cglennon924**

### CODE REVIEW

File: api.js [hunk 1]
```diff
@@ -36,91 +37,92 @@ $(function() {
         }
     });
 
-    $("#submit").on("click", function(){
 
-        $( "#clear" ).click(function() {
-            $( "#output" ).empty();
-
```
Comment (cag-martinez) [inline api.js:43]: these lines are repeated, we dont need them, only one clear button.

File: api.js [hunk 2]
```diff
@@ -36,91 +37,92 @@ $(function() {
...
         })
-        .done(function() {
-            console.log("success");
-        })
-        .fail(function() {
-            console.log("error");
-        })
-        .always(function() {
-            console.log("complete");
-        });
```
Comment (cag-martinez) [inline api.js:71]: these lines are not needed, they are also repeated

File: api.js [hunk 3]
```diff
@@ -36,91 +37,92 @@ $(function() {
...
             method: "GET",
             contentType: "application/json; charset=utf-8",
             async: false,
             dataType: "json",
-            success: function(data, status, jqXHR) {
+            success: function (data, status, jqXHR) {
                 $("#output").html();
-                for(var i=0; i < data[1].length-1; i++) {
-                    $("#output").append("<div><div class='btn-primary'><a href=" +data[3][i]+ "><h2>" +data[1][i]+ "</h2>" + "<p>" + data[2][i] + "</p></a></div></div>");
-                console.log(data);
```
Comment (cag-martinez) [inline api.js:59]: repeated

File: api.js [hunk 4]
```diff
@@ -36,91 +37,92 @@ $(function() {
...
+        var queryURL = "https://en.wikipedia.org/w/api.php?action=opensearch&search=" + searchTerm + "&format=json&callback=?";
+
 
         $.ajax({
             url: queryURL,
             method: "GET",
             contentType: "application/json; charset=utf-8",
             async: false,
             dataType: "json",
-            success: function(data, status, jqXHR) {
```
Comment (cag-martinez) [inline api.js:55]: remove

File: api.js [hunk 5]
```diff
@@ -36,91 +37,92 @@ $(function() {
...
-            $( "#output" ).empty();
-            
+    $("#submit").on("click", function () {
+
+        $("#clear").click(function () {
+            $("#output").empty();
         });
-    
-        var artist = $("textarea").val();
-        var queryURL = "https://en.wikipedia.org/w/api.php?action=opensearch&search="+ artist +"&format=json&callback=?";
```
Comment (cag-martinez) [inline api.js:47]: remove this **var** and **queryURL** the one bellow is the one we will be working with

**Review by cag-martinez — COMMENTED** [review]
I left a few notes one some code that is repeated or we do not need.

**Review by Khamari13 — APPROVED** [review]

**Pull request merged**